commit/merkleroot/rmn/crypto_test.go

```diff
@@ -0,0 +1,133 @@
+package rmn
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-ccip/commit/merkleroot/rmn/rmnpb"
+)
+
+var (
+	empty32ByteArr [32]byte
+	empty20ByteArr [20]byte
+)
+
+func Test_verifyObservationSignature(t *testing.T) {
```

Comment:
I think you can re-organize these test cases a bit better, maybe put the required values in an `args` struct and have two different test cases? The actual code that is running seems to be the same for both.

Reply:
done